src/lib/terra-components.module.ts

```diff
@@ -40,14 +41,16 @@ import { HttpModule } from '@angular/http';
     ],
     declarations:    [
         ...components,
-        ...directives
+        ...directives,
+        SelectSortPipe
     ],
     entryComponents: [
         ...exportedComponents
     ],
     exports:         [
         ...exportedComponents,
-        ...directives
+        ...directives,
+        SelectSortPipe
```

Comment:
Ich fände es gut, wenn es auch hier gleich eine Sammlung von Pipes geben würde. Damit wir hier in Zukunft einfach erweitern können.